packages/core/src/utils/theme.ts

```diff
@@ -37,18 +25,9 @@ export const setElementAttrs = (
     element.setAttribute(`data-theme`, themeName);
     element.setAttribute(`data-color-mode`, modeName);
 
-    // Set default properties for all components to inherit
-    setElementStyle(element, {
-      colorScheme,
-      backgroundColor: theme.colors.backgroundColor,
-      accentColor: theme.colors.secondary,
-      color: theme.colors.secondary,
-      fontSize: theme.typography.body.fontSize,
-      fontWeight: theme.typography.body.fontWeight,
-      lineHeight: theme.typography.body.lineHeight,
-      letterSpacing: theme.typography.body.letterSpacing,
-      fontFamily: theme.fontFamily.body,
```

Comment:
looks like we lost `fontFamily` in the refactor; is this intentional?

Reply:
No, it wasn't.